static/js/services/db.js

```diff
@@ -134,8 +134,7 @@ var utils = require('kujua-utils'),
               .changes({
                 live: true,
                 since: 'now',
-                doc_ids: [ '_design/medic' ],
-                timeout: 1000 * 60 * 60
```

Comment:
Did you need to remove this?

(FWIW it's fine, just interested in the reasoning for this and the other similar change)